docs/requirements.txt

```diff
@@ -0,0 +1,2 @@
+Sphinx>=5,<5.1
+sphinx_rtd_theme
```

Comment:
We should probably be pinning transitive dependencies here: https://docs.readthedocs.io/en/latest/guides/reproducible-builds.html#pinning-transitive-dependencies

Reply:
I was unsure about what to do with especially `sphinx_rtd_theme` since we release updates... how can we ensure that updates propagate to projects if they have all pinned `sphinx_rtd_theme`? I sense that the answer is probably a reference to a previous discussion :smile:

Reply:
We should pin all the dependencies, as noted in that doc. Reproducible builds that don't break randomly are more important than any theoretical new feature for users, especially for less advanced users who don't understand what is happening.

Reply:
In this case, I think that we should have dependabot active on the example project itself and accept the work of keeping our examples with latest pinned requirements... as much as possible. Otherwise, users of the repo also be able to notice the PRs from dependabot which is a good Plan B :+1:

Reply:
I went as far as to use pip-tools now. Need to figure out to communicate the pip-tools usage + add dependabot.

Reply:
All requirements are pinned now btw.